manifest.json

```diff
@@ -32,7 +32,6 @@
     }
   },
   "scripts": {
-    "prereleasy": "bash lint.sh",
     "postreleasy": "vtex publish --verbose"
```

Comment:
I think it's a good idea to remove this `postreleasy` script, as this would conflit with the automatic releases made by our bot.
If you try to run this in your terminal, you would get and error saying that your app was already published, because the commits `releasy` pushes to GitHub trigger the bot to publish the app

Reply:
Yeah, I can remove in the next release